### PR TITLE
Remove ev3dev-lang and autogen

### PR DESCRIPTION
**Pull request by dwalton76**
```
robot@evb[ev3dev-lang-python]# git grep autogen
docs/conf.py:# autogenerated file.
docs/conf.py:# The language for content autogenerated by Sphinx. Refer to documentation
robot@evb[ev3dev-lang-python]# 
```

### CODE REVIEW

**Comment by ddemidov** [comment]
We should also drop `templates` directory.

File: docs/sensors.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ The classes derive from :py:class:`Sensor` and provide helper functions
 specific to the corresponding sensor type. Each of the functions makes
 sure the sensor is in the required mode and then returns the specified value.
 
-.. ~autogen doc-special-sensor-classes
+..
```
Comment (ddemidov) [inline docs/sensors.rst:21]: One of the reasons I liked autogen: I bet next time we change something in a special sensor, we forget to document it :).

Comment (dwalton76) [reply docs/sensors.rst:21]: True...there is always a tradeoff :)

**Review by ddemidov — APPROVED** [review]

**Comment by jabrena** [comment]
Hi @dwalton76, @ddemidov,

why are removing the ev3dev-lang in the project?

Juan Antonio

**Review by WasabiFan — APPROVED** [review]

**Comment by ddemidov** [comment]
> why are removing the ev3dev-lang in the project?

During #351 we decided that the API is now stable enough so that autogen has more disadvantages than advantages.

**Comment by jabrena** [comment]
I agree with you @ddemidov, the API delivered with `ev3dev` is robust and stable.

Cheers

**Pull request merged**